packages/code/src/Code.tsx

```diff
@@ -80,6 +94,86 @@ function getWrapperVariantStyle(variant: Variant): string {
   `;
 }
 
+function getSidebarVariantStyle(variant: Variant): string {
+  const colors = variantColors[variant] ?? variantColors[Variant.Light];
+
+  switch (variant) {
+    case Variant.Light:
+      return css`
+        border-color: ${colors[1]};
+        background-color: white;
+      `;
+    case Variant.Dark:
+      return css`
+        border-color: ${colors[1]};
+        background-color: ${colors[1]};
+      `;
+  }
+}
+
+function getCopyButtonStyle(variant: Variant, copied: boolean): string {
+  if (copied) {
+    return css`
+      align-self: center;
+      color: ${uiColors.white};
+      background-color: ${uiColors.green.base};
+    `;
+  }
+
+  if (variant === Variant.Dark) {
+    return css`
+      align-self: center;
+      color: ${uiColors.gray.base};
+      background-color: ${uiColors.gray.dark3};
+    `;
+  }
+
+  return css`
```

Comment:
Nice happy-pathing! There's a bit of style repetition here that we can clean up using `cx`:

```js
  const baseStyle = css`
    align-self: center;
    color: ${uiColors.gray.base};
  `;

  if (copied) {
    return cx(baseStyle, css`
      color: ${uiColors.white};
      background-color: ${uiColors.green.base};
    `);
  }

  if (variant === Variant.Dark) {
    return cx(baseStyle, css`
      background-color: ${uiColors.gray.dark3};
    `);
  }

  return baseStyle;
```